java/src/IceGridGUI/src/main/java/com/zeroc/IceGridGUI/LiveDeployment/Node.java

```diff
@@ -269,16 +269,22 @@ protected String getDefaultFileName()
     Utils.ExpandedPropertySet expand(PropertySetDescriptor descriptor, String applicationName, Utils.Resolver resolver)
     {
         Utils.ExpandedPropertySet result = new Utils.ExpandedPropertySet();
-        result.references = new Utils.ExpandedPropertySet[descriptor.references.length];
+        java.util.ArrayList<Utils.ExpandedPropertySet> references = new java.util.ArrayList<Utils.ExpandedPropertySet>();
 
-        int i = 0;
         for(String ref : descriptor.references)
         {
-            result.references[i++] = expand(findNamedPropertySet(resolver.substitute(ref), applicationName),
-                                            applicationName, resolver);
+            PropertySetDescriptor resolved = findNamedPropertySet(resolver.substitute(ref), applicationName);
+            if (resolved == null)
```

Comment:
I think it would make more sense to render it as an empty property set and not log a warning.
It sounds like the registry is happy to treat this unknown property set reference as an empty property set.

Reply:
I removed the warning